### PR TITLE
Add visual servo teach / move Objectives

### PR DESCRIPTION
**Pull request by marioprats**
Adds a demo Visual Servo Objective in simulation.
Requires https://github.com/PickNikRobotics/moveit_studio/pull/5999

Used in the tutorial in https://github.com/PickNikRobotics/moveit_studio/pull/6251.

Closes https://github.com/PickNikRobotics/moveit_studio/issues/5922.

### CODE REVIEW

**Comment by pac48** [comment]
I tried testing this, but currently I get an error from the UI:
```
Behavior: objective_server_node. Error: Failed to create behavior tree for Objective `Visual Servo To Reference`. Reason: BehaviorTree error while creating tree: Missing manifest for element_ID: ServoTowardsPose. It shouldn't happen. Please report this issue.
```
and form the backend:
```
moveit_studio_ws-agent_bridge-1  | [objective_server_node_main-7] [ERROR] [1706908229.370807930] [Logger]: Behavior: objective_server_node. Error: Failed to create behavior tree for Objective `Visual Servo To Reference`. Reason: BehaviorTree error while creating tree: Missing manifest for element_ID: ServoTowardsPose. It shouldn't happen. Please report this issue.
moveit_studio_ws-agent_bridge-1  | [rosbridge_websocket-21] [INFO] [1706908229.370855175] [rosbridge_websocket]: [Client 9fe91a1f-97f0-4b41-8194-8df5088fa76f] Subscribed to /preview_solution
moveit_studio_ws-agent_bridge-1  | [move_group-1] [WARN] [1706908229.374874709] [move_group.moveit.planning_scene_monitor]: Unable to transform object from frame 'apriltag_36h11_1' to planning frame'world' (Lookup would require extrapolation into the past.  Requested time 100.765000 but the earliest data is at time 152.355000, when looking up transform from frame [apriltag_36h11_1] to frame [world])
moveit_studio_ws-agent_bridge-1  | [move_group-1] [WARN] [1706908229.374902386] [move_group.moveit.planning_scene_monitor]: Unable to transform object from frame 'apriltag_36h11_2' to planning frame'world' (Lookup would require extrapolation into the past.  Requested time 100.765000 but the earliest data is at time 152.355000, when looking up transform from frame [apriltag_36h11_2] to frame [world])
moveit_studio_ws-agent_bridge-1  | [move_group-1] [WARN] [1706908229.374908796] [move_group.moveit.planning_scene_monitor]: Unable to transform object from frame 'apriltag_36h11_3' to planning frame'world' (Lookup would require extrapolation into the past.  Requested time 100.765000 but the earliest data is at time 152.355000, when looking up transform from frame [apriltag_36h11_3] to frame [world])
moveit_studio_ws-agent_bridge-1  | [move_group-1] [WARN] [1706908229.374914576] [move_group.moveit.planning_scene_monitor]: Unable to transform object from frame 'avg_pose' to planning frame'world' (Lookup would require extrapolation into the past.  Requested time 100.765000 but the earliest data is at time 152.355000, when looking up transform from frame [avg_pose] to frame [world])

```

**Comment by marioprats** [comment]
@pac48 I think there was an issue with the averaging Behavior, but it should be fixed now.
Although I'm not sure how that could cause it to move upwards :thinking: 
Anyway this is ready if you want to test again. Thank you!

**Comment by pac48** [comment]
> @pac48 I think there was an issue with the averaging Behavior, but it should be fixed now. Although I'm not sure how that could cause it to move upwards 🤔 Anyway this is ready if you want to test again. Thank you!

I think the problem is that the reference point was in an invalid place. Maybe there was too much error in the tag estimation, which put it out of the robot's reach. It is working for me now.

File: src/picknik_ur_gazebo_config/objectives/visual_servo_to_reference.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Visual Servo To Reference">
+  <!-- ////////// -->
+  <BehaviorTree ID="Visual Servo To Reference" _description="Move in closed loop to a pose relative to an AR Tag" _favorite="true" _hardcoded="false">
```
Comment (pac48) [inline src/picknik_ur_gazebo_config/objectives/visual_servo_to_reference.xml:4]:
```suggestion
  <BehaviorTree ID="Visual Servo To Reference" _description="Move in closed loop to a pose relative to an AprilTag" _favorite="true" _hardcoded="false">
```

**Review by pac48 — APPROVED** [review]

**Review by moriarty — APPROVED** [review]

**Pull request merged**